<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>details</class>
 <widget class="QWidget" name="details">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>216</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="SizeLabel">
       <property name="text">
        <string>Size:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="KComboBox" name="sizeCombo"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="RateLabel">
       <property name="text">
        <string>Refresh Rate:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="KComboBox" name="rateCombo"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="OrientationLabel">
       <property name="text">
        <string>Orientation:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="KComboBox" name="orientationCombo"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="ReflectionLabel">
       <property name="text">
        <string>Reflection:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="KComboBox" name="reflectionCombo"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="PositionLabel">
       <property name="text">
        <string>Position:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="KComboBox" name="posCombo1"/>
       </item>
       <item row="0" column="1">
        <widget class="KComboBox" name="posCombo2"/>
       </item>
       <item row="1" column="0">
        <widget class="KIntNumInput" name="posxNumInput"/>
       </item>
       <item row="1" column="1">
        <widget class="KIntNumInput" name="posyNumInput"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>202</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KIntNumInput</class>
   <extends>QWidget</extends>
   <header>knuminput.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
